massdash/server/PeakPickingServer.py

```diff
@@ -35,7 +35,7 @@ def __init__(self, peak_picking_settings: PeakPickingUISettings, chrom_plot_sett
 
     def perform_osw_pyprophet_peak_picking(self, xic_data: SqMassLoader, transition_list_ui: Literal['ExtractedIonChromatogramAnalysisUI', 'RawTargetedExtractionAnalysisUI']):
```

Comment:
I am having difficulties with this method in the GUI could you please look into this? I think that the loader object for the raw data is not currently being passed which causes loadTransitionGroupFeature to fail. Not sure what the best way to solve this without just adding more arguments

Reply:
You're getting this error?

> AttributeError: 'NoneType' object has no attribute 'loadTransitionGroupFeature'

Reply:
yes thats the error. I think it is because there is no XIC data/loader object passed

Reply:
I passed the mzml loader object now instead to get the feature data from the results file. I think I initially used the sqMass workflow that's why it failed.